easybuild/tools/github.py

```diff
@@ -896,7 +896,10 @@ def check_github():
         else:
             check_res = "OK"
     elif github_user:
-        check_res = "FAIL (unexpected exception: %s)" % push_err
+        if sys.modules['git'].__version__ < "1.0.0":
```

Comment:
please use `LooseVersion` when comparing versions

also, just use `git.__version__`?

```
if LooseVersion(git.__version__) < LooseVersion('1.0'):
```

Did you verify whether things (should) work with GitPython 1.0 and up?